### ISSUE TITLE
#<TypeError: wrong argument type Hash (expected Proc)>

### ISSUE THREAD

**Issue by stereobooster** [chanks/que#191]
This error comes from this line:

```ruby
CAST_PROCS[114] = -> (value) { JSON_MODULE.load(value, create_additions: false) }
```

Because [JSON.load](http://ruby-doc.org/stdlib-2.0.0/libdoc/json/rdoc/JSON.html#method-i-load) second argument is proc.

**Comment by chanks** [comment]
Weird, I wonder why this didn't turn up in the specs. Will look into it...

**Comment by chanks** [comment]
Alright, I just pushed a change to master that simplifies things by removing support for MultiJson. This should fix your issue - can you try master and let me know if it works for you?

**Comment by stereobooster** [comment]
@chanks yep that works

**Comment by chanks** [comment]
Oops, forgot to close this. I released 0.13.0 yesterday.
